File: Server/Tokenizer_V1/Tokenizer_V1/Requests/Companies/SearchCompaniesReq.cs
using Clinic_V2._0.Paging;
using Microsoft.AspNetCore.Http;

namespace Tokenizer_V1.Requests.Companies
{
    public class SearchCompaniesReq
    {
        public int? Id { set; get; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string Zip { get; set; }
        public string Country { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public string Website { get; set; }
        public int? CompanyTypeId { get; set; }

        public PagingParams pagingParams { set; get; }

    }
}

File: Client/v1/src/src/Requests/Companies/SearchCompaniesReq.js
class SearchCompaniesReq {
    constructor() {
        this.id = undefined;
        this.name = undefined;
        this.description = undefined;
        this.address = undefined;
        this.city = undefined;
        this.zip = undefined;
        this.country = undefined;
        this.phone = undefined;
        this.email = undefined;
        this.website = undefined;
        this.companyTypeId = undefined;
        this.pagingParams = {
            pageNumber: 1,
            pageSize: 10
        }
    }
}
export default SearchCompaniesReq;